### PR TITLE
 feat(argo-cd): Add resources to copyutil initContainer

### LINKED ISSUE #1087: Repo-servers copyutil initContainer does not set resources
### Describe the bug

The repo server has an InitContainer called "copyutil". This container does not set its resources.

### Related helm chart

argo-cd

### To Reproduce

Just render a template of the chart. There are no ressources set for the copyutil initContainer.

### Expected behavior

The container makes use of resources predefinded in in the repo-server values.
The same initContainer as part of the dex deployment is equipped with resources defined in `.Values.dex.resources`

### Screenshots

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by shaardie**
Resolves #1087

---
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#versioning)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo-cd/README.md [hunk 1]
```diff
@@ -316,6 +316,8 @@ NAME: my-release
 | repoServer.clusterRoleRules.rules | list | `[]` | List of custom rules for the Repo server's Cluster Role resource |
 | repoServer.containerPort | int | `8081` | Configures the repo server port |
 | repoServer.containerSecurityContext | object | `{}` | Repo server container-level security context |
+| repoServer.copyutil | object | `{"resources":{}}` | Init Container to copy argocd binary |
```
Comment (mbevc1) [inline charts/argo-cd/README.md:319]: Not a blocker, but probably we could drop this line due being implicitly documented in next one? :thinking:

File: charts/argo-cd/values.yaml [hunk 2]
```diff
@@ -1535,6 +1535,17 @@ repoServer:
   #     - list
   #     - watch
 
+  # -- Init Container to copy argocd binary
```
Comment (mkilchhofer) [inline charts/argo-cd/values.yaml:1538]: Drop the `--` from this line and rerun helm-docs.

**Review by mkilchhofer — CHANGES_REQUESTED** [review]

File: charts/argo-cd/values.yaml (hunk 2 above)
Comment (shaardie) [reply charts/argo-cd/values.yaml]: Done.

File: charts/argo-cd/README.md (hunk 1 above)
Comment (shaardie) [reply charts/argo-cd/README.md]: I removed it (automatically with `helm-docs`)

**Review by mbevc1 — APPROVED** [review]

**Review by mkilchhofer — APPROVED** [review]

**Comment by mbevc1** [comment]
Thanks for the contribution!

**Pull request merged**